docs/getting_started.rst

```diff
@@ -317,6 +317,44 @@ You can do this automatically with the ``--autodesktop`` flag:
     $ aqt install-qt linux desktop 6.2.0 wasm_32 -m qtcharts qtnetworkauth --autodesktop
 
 
+Installing Docs
+---------------
+
+The :ref:`aqt list-doc <list-doc command>` command lists documentation
+archives for a given version by default:
+
+.. code-block:: console
+
+    $ aqt list-tool mac 6.6.1
+    qdoc qmake qt5 qtassistant qtcmake qtconcurrent qtcore qtdbus qtdesigner qtdistancefieldgenerator qtdoc qtgui qthelp qtlabsplatform qtlinguist qtnetwork qtopengl qtplatformintegration qtprintsupport qtqml qtqmlcore qtqmlmodels qtqmltest qtqmlworkerscript qtqmlxmllistmodel qtquick qtquickcontrols qtquickdialogs qtsql qtsvg qttestlib qtuitools qtwaylandcompositor qtwidgets qtxml
+
+This is what will be installed by default, you can select a subset of archives
+with ``--archives``.
+``--modules`` will list additional documentation modules that can be installed:
```

Comment:
```suggestion
This is what will be installed by default when you run ``aqt install-doc mac 6.6.1`` 
without any other flags or arguments. 
You can select a subset of archives with the ``--archives`` flag.
The ``--modules`` flag will list additional documentation modules that can be installed:
```
This is more of a writing-style / grammar suggestion. IMHO it's worthwhile to be as clear as possible here. I'm certainly not the best writer around, but I think we can try to make this a little better.

Mozi is pretty good at this stuff. @pzhlkj6612, any suggestions?